### PR TITLE
Fixingissues

### PR DESCRIPTION
**Pull request by sahandfarahani**

### CODE REVIEW

File: src/utils/deleteFileFromPath/deleteFileFromPath.ts [hunk 1]
```diff
@@ -13,15 +13,15 @@ const DELETION_INCREMENT = 1;
 
 const recursivelySetDeletedOrder = (
   treeCore: Core[],
-  newDeletedOrder: number,
+  newDeletedOrder: number
 ): void => {
   for (let index = 0; index < treeCore.length; index += 1) {
     if (treeCore[index].deletedOrder === -1) {
```
Comment (ralph-dev) [inline src/utils/deleteFileFromPath/deleteFileFromPath.ts:19]: -1 should be a constant. Try to fix other peoples code as you go for this project for learning purposes.

File: src/utils/getFileTypeFromPath/getFileTypeFromPath.ts [hunk 2]
```diff
@@ -1,58 +1,36 @@
...
-export const getFileTypeFromPath = (path: string): FileType => {
-  // Count amount of periods in the path
-  const numberOfDotsInPath = (path.match(/\./g) || []).length;
-  // Breaks the path up at the forward slashes
-  const forwardSlashPathFilter = path.split('/');
-
-  const configNumberOfDots = 2; // number of dots in a config file path
-  const fileNumberOfDots = 1; // number of dots in a regular file path
-  const folderNumberOfDots = 0; // number of dots in a folder path
+export const getFileTypeFromPath = (
```
Comment (ralph-dev) [inline src/utils/getFileTypeFromPath/getFileTypeFromPath.ts:10]: JSDOC

Comment (sahandfarahani) [reply src/utils/getFileTypeFromPath/getFileTypeFromPath.ts:10]: @ralph-dev Isn't the notes after import JSDOC?

Comment (ralph-dev) [reply src/utils/getFileTypeFromPath/getFileTypeFromPath.ts:10]: Add JSDOC for getFileTypefromPath function
/**
* Get's the file type by analyzing the number of "."'s etc
*/

**Comment by sahandfarahani** [comment]
> @sahandfarahani
> 
> LGTM -- one of the tests (undoDeletionsTest) doesn't pass though :/

Thank You, I'll get into it

**Comment by sahandfarahani** [comment]
> @sahandfarahani
> 
> LGTM -- one of the tests (undoDeletionsTest) doesn't pass though :/
 its funny... it looks like the problem is deletefrompath but when I try the same process inside the project it works correctly

**Pull request merged**